omnibus/config/software/oc_id.rb

```diff
@@ -47,9 +47,9 @@
          " --with-xml2-config=#{install_dir}/embedded/bin/xml2-config" \
          " --with-xslt-config=#{install_dir}/embedded/bin/xslt-config"
 
+  bundle config set --local without development test doc
```

Comment:
In this context, `bundle` is a method, so you need to pass the rest in as a string. 

```
bundle "config set --local without development test doc"
```

Reply:
Sure, I'll make changes accordingly.